hack/jenkins/e2e.sh

```diff
@@ -612,26 +612,76 @@ case ${JOB_NAME} in
     : ${E2E_UP:="false"}
     : ${E2E_TEST:="true"}
     : ${E2E_DOWN:="false"}
-    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-cluster"}
-    NUM_MINIONS=3
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=Skipped.*Cluster\supgrade.*upgrade-master"}
+    : ${NUM_MINIONS:=3}
```

Comment:
Do you know why we're running these on clusters of 3, as opposed to just using the default, (not sure what that is—2?)?